<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pogadeSettings</class>
 <widget class="QDialog" name="pogadeSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>143</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Pogade Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="3" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Polca Reader:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QPushButton" name="buttonPRSelect">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="linePR"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_3">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Polca Init:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="linePI">
       <property name="enabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="buttonPISelect">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Polca Apply:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="linePA"/>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="buttonPASelect">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Select Language:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboLanguage"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="buttonApply">
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonOK">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonApply</sender>
   <signal>clicked()</signal>
   <receiver>pogadeSettings</receiver>
   <slot>doApply()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>177</x>
     <y>245</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonOK</sender>
   <signal>clicked()</signal>
   <receiver>pogadeSettings</receiver>
   <slot>doOK()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>349</x>
     <y>245</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonCancel</sender>
   <signal>clicked()</signal>
   <receiver>pogadeSettings</receiver>
   <slot>doCancel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>263</x>
     <y>245</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonPRSelect</sender>
   <signal>clicked()</signal>
   <receiver>pogadeSettings</receiver>
   <slot>chooseToolPR()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>349</x>
     <y>53</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonPISelect</sender>
   <signal>clicked()</signal>
   <receiver>pogadeSettings</receiver>
   <slot>chooseToolPI()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>349</x>
     <y>18</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>71</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonPASelect</sender>
   <signal>clicked()</signal>
   <receiver>pogadeSettings</receiver>
   <slot>chooseToolPA()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>349</x>
     <y>41</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>71</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>doApply()</slot>
  <slot>doCancel()</slot>
  <slot>doOK()</slot>
  <slot>chooseToolPR()</slot>
  <slot>chooseToolPI()</slot>
  <slot>chooseToolPA()</slot>
 </slots>
</ui>
